duckduckgo/iOS
# Privacy Protection: Score Card

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/457416080701391/f
CC:

**Description**:

The privacy protection score card.

**Steps to test this PR**:
1. Visit various sites and ensure the score card reflects the information on the initial screen
1. Ensure the score card updates in real time, like the initial screen


###### Reviewer Checklist:


###### PR DRI Checklist:
